motoman_driver/src/joint_trajectory_streamer.cpp

```diff
@@ -380,6 +399,8 @@ bool MotomanJointTrajectoryStreamer::send_to_robot(const std::vector<SimpleMessa
 void MotomanJointTrajectoryStreamer::streamingThread()
 {
   int connectRetryCount = 1;
+  bool is_connected = false;
+  bool is_msg_sent = false;
```

Comment:
I'd like to do that, but I believe we can't. At least `is_connected` needs to be in the scope which contains the `switch (this->state_)`. So it can be moved to be inside the outer `while (ros::ok())`, but that doesn't really change things I believe.